src/stru_multi/4C_stru_multi_microstatic.hpp

```diff
@@ -133,12 +133,14 @@ namespace MultiScale
     void output(Core::IO::DiscretizationWriter& output, const double time, const int istep,
         const double dt);
 
+    void runtime_output(
```

Comment:
Can you please add a short description, what output quantities are written here?